task_templates/pipelines/python3_anomaly_detection/custom.py

```diff
@@ -1,40 +1,112 @@
+"""
+    In this example we see how to create an anomaly detection (i.e. unsupervised) pipeline
+
+    Note: in our score hook we use the same column name as regression, i.e. columns=['Predictions']
```

Comment:
instead of here, shall we describe it where we describe what the score hook returns?